### ISSUE TITLE
app quits immediately if setAccessibilitySupportEnabled is called

### ISSUE THREAD

**Issue by marisademeglio** [electron/electron#16256]
* Output of `node_modules/.bin/electron --version`: v4.0.0
* Operating System (Platform and Version): macOS 10.14.2

* Output of `node_modules/.bin/electron --version` on last known working Electron version (if applicable): 2.0.7

### Expected behavior

I should be able to call `app.setAccessibilitySupportEnabled(true)` during initialization, and the application should start and stay open.

### Actual behavior

The application starts and exits immediately if `app.setAccessibilitySupportEnabled(true)` is called during initialization. Same for passing in `false`.

### How to reproduce

Here is a simple project that shows this error. The application behaves as expected (does not exit immediately) when this line is commented out:
https://github.com/marisademeglio/electron4-setA11yTreeEnabled/blob/master/src/main/main.js#L27

  $ git clone https://github.com/marisademeglio/electron4-setA11yTreeEnabled.git
  $ yarn install
  $ yarn start

**Comment by codebytere** [comment]
i just ran this in electron fiddle on `v4.0.0` with no issues:
```
const {app, BrowserWindow, systemPreferences} = require('electron')

let mainWindow

function createWindow () {
  mainWindow = new BrowserWindow({width: 800, height: 600})

  mainWindow.loadURL('https://github.com')

  app.setAccessibilitySupportEnabled(true)

  mainWindow.on('closed', () => mainWindow = null)
}

app.on('ready', createWindow)

// Quit when all windows are closed.
app.on('window-all-closed', () => {
  if (process.platform !== 'darwin') app.quit()
})

app.on('activate', () => {
  if (mainWindow === null) createWindow()
})
```
do you have a more minimal example that shows your issue?

**Comment by marisademeglio** [comment]
Here, it seems to depend on the order: 
https://gist.github.com/969873022f427dfa3746360e399ccfe9

If I put it in createWindow() as you did, it works. But if I put it before the window is created, it exits immediately.
